test/yamls/v1beta1/pipelineruns/pipelinerun.yaml

```diff
@@ -89,18 +89,6 @@ spec:
     # makes it explicit that it needs to run as root.
     securityContext:
       runAsUser: 0
-  - name: write-digest
```

Comment:
Tried deleting these steps, but I have been wondering if this would make the whole test case meaningless without the image digest?

Reply:
I reviewed the examples and testes that are deleted so far as part of this PR. I would have preferred to replace the `image` resource with something equivalent but I am not sure what is that equivalent. 😞